testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/HydraIdentityProvider.java

```diff
@@ -13,6 +13,7 @@
  */
 package io.trino.tests.product.launcher.env.common;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
>  Make HydraIdentityProvider env extender Trino nodes friendly


maybe cmt msg "Configure worker if present in HydraIdentityProvider env extender"